### DISCUSSION TITLE
Which version of Eva is this mod compatible with?

### DISCUSSION THREAD

**Discussion by pasha4ur** [majarspeed/Voron-EVA-conversion#8] [Q&A]
Hello.

Thank you for your work.

Is this mod compatible with Eva 3?

**Comment by majarspeed** [comment]
No it is not.

**Reply by CD1010** [reply]
Is it 2.4?

**Comment by pasha4ur** [comment]
Do you have any plans to update it to version 3? This will be great.

Voron's printhead isn't very good.

**Reply by majarspeed** [reply]
I looked at it and the weight and it he size make it less than ideal. I am leaving it based on 2.4 for now.
